src/provider/common/Injectables.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Declare all injectable types/values here.
+ * 
+ * This enum determines the keys that can be used within `@inject` tags. The mapping of these tags to default concrete types is defined in 
+ * `inversify.config.ts`. These mappings can be programmatically overridden by calling the methods of the {@link Injector} util.
```

Comment:
Oops. This is referring to `Injector.ts`, the file got renamed along the way when it became a util rather than using inversify directly. Cuts down on boilerplate a bit, since inversify can do a fair bit more than what we're using it for.

Will update the comment with the right filename.